server.js

```diff
@@ -21,7 +12,47 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(expressListEndpoints(app));
+});
```

Comment:
when you list the endpoints you can add more info to document your API, for example you can filter the data by queries like `year`, you could add something like this:

```javascript
app.get("/", (req, res) => {
  const endpoints = expressListEndpoints(app);

  // Add query parameters to the endpoints documentation
  endpoints.forEach((endpoint) => {
    if (endpoint.path === "/books") {
      endpoint.queryParameters = {
        year: "Filter by release year",
      };
    }
  });

  res.json(endpoints);
});
``` 
This way you document the full usage of your API 👍